plugins/resource-recommendations.yml

```diff
@@ -0,0 +1,26 @@
+plugin:
+# Recommends requests and limits for CPU and memory
```

Comment:
We should precise the dependencies that must be present on the cluster and the user shell.
Also we should have an author: xxx specification in the docs.

Reply:
Got it.